src/core/mount.c

```diff
@@ -2106,11 +2124,27 @@ static void mount_enumerate(Manager *m) {
                 }
 
                 (void) sd_event_source_set_description(m->mount_event_source, "mount-monitor-dispatch");
+
+                if (!MANAGER_IS_SYSTEM(m)) {
+                        r = sd_event_add_post(m->event, &m->mount_enumerate_source, mount_enumerate_late, m);
+                        if (r < 0) {
+                                log_error_errno(r, "Failed to add late mount enumerate source: %m");
+                                goto fail;
+                        }
+
+                        r = sd_event_source_set_priority(m->mount_enumerate_source, SD_EVENT_PRIORITY_IDLE);
```

Comment:
idle seems very low for this. also, please use the EVENT_PRIORITY_XYZ enum that now is add the end of manager.h